metrics/web/lib/auth/presentation/util/email_validation_util.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:metrics/auth/presentation/model/email_validation_error_message.dart';
+import 'package:metrics_core/metrics_core.dart';
+
+class EmailValidationUtil {
+  /// Validates the given [value] as an email.
+  ///
+  /// Returns an error message if the [value] is not a valid email,
+  /// otherwise returns null.
+  static String validateEmail(String value) {
```

Comment:
```suggestion
  static String validate(String value) {
```
just to simplify as we'll still use class name?